inference-engine/tests/functional/plugin/cpu/single_layer_tests/interpolate.cpp

```diff
@@ -640,7 +784,17 @@ const auto interpolateCornerCases = ::testing::Combine(
 
 INSTANTIATE_TEST_SUITE_P(smoke_Interpolate_corner_Layout_Test, InterpolateLayerCPUTest,
         ::testing::Combine(
-            interpolateCornerCases,
+            interpolateCornerCases_Smoke,
+            ::testing::ValuesIn(shapeParams4D_corner),
+            ::testing::Values(ElementType::f32),
+            ::testing::ValuesIn(filterCPUInfoForDevice()),
+            ::testing::ValuesIn(interpolateFusingParamsSet),
+            ::testing::ValuesIn(filterAdditionalConfig())),
+    InterpolateLayerCPUTest::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(Interpolate_corner_Layout_Test, InterpolateLayerCPUTest,
```

Comment:
the same as smoke_Interpolate_corner_Layout_Test for corner case

Reply:
Fixed in: 06054aceba7ccb9727eac838c9dd54798ca7b60e